fails to compile on freebsd 14-current

Compiling wayland-commons v0.28.6
   Compiling wayland-commons v0.29.4
   Compiling jpeg-decoder v0.1.22
   Compiling tiff v0.6.1
   Compiling wayland-cursor v0.28.6
   Compiling thiserror-impl v1.0.30
   Compiling futures-macro v0.3.19
   Compiling enumflags2_derive v0.6.4
   Compiling serde_repr v0.1.7
   Compiling derivative v2.2.0
   Compiling tokio-macros v1.7.0
   Compiling image v0.23.14
   Compiling wayland-cursor v0.29.4
   Compiling tokio v1.15.0
   Compiling thiserror v1.0.30
   Compiling futures-executor v0.3.19
   Compiling futures v0.3.19
   Compiling smithay-client-toolkit v0.14.0
   Compiling smithay-clipboard v0.6.5
   Compiling enumflags2 v0.6.4
   Compiling toml v0.5.8
   Compiling proc-macro-crate v1.1.0
   Compiling proc-macro-crate v0.1.5
   Compiling zvariant_derive v2.10.0
   Compiling zbus_macros v1.9.2
   Compiling zvariant v2.10.0
   Compiling zbus v1.9.2
error[E0432]: unresolved import `nix::sys::socket::sockopt::PeerCredentials`
   --> /home/buckbucks/.cargo/registry/src/github.com-1ecc6299db9ec823/zbus-1.9.2/src/connection.rs:163:44
    |
163 |         use nix::sys::socket::{getsockopt, sockopt::PeerCredentials};
    |                                            ^^^^^^^^^^^^^^^^^^^^^^^^ no `PeerCredentials` in `sys::socket::sockopt`

error[E0432]: unresolved import `nix::sys::socket::sockopt::PeerCredentials`
   --> /home/buckbucks/.cargo/registry/src/github.com-1ecc6299db9ec823/zbus-1.9.2/src/azync/connection.rs:180:44
    |
180 |         use nix::sys::socket::{getsockopt, sockopt::PeerCredentials};
    |                                            ^^^^^^^^^^^^^^^^^^^^^^^^ no `PeerCredentials` in `sys::socket::sockopt`

For more information about this error, try `rustc --explain E0432`.
error: failed to compile `kickoff v0.4.5`, intermediate artifacts can be found at `/tmp/cargo-installzgmK7Q`

Caused by:
  could not compile `zbus` due to 2 previous errors

error on cargo install

> error[E0432]: unresolved import nix::sys::socket::sockopt::PeerCredentials

zbus < 2.0.0-beta.4 doesn't support non-Linux. If you can't upgrade yet try [patching the crates](https://github.com/freebsd/freebsd-ports/blob/5b78e25e4b82/shells/starship/files/patch-zbus).

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -31,9 +31,7 @@ simplelog = "0.11.1"
 tokio = { version = "1.9.0", features = ["full"] }
 x11-keysymdef = "0.2.0"
 fontconfig = "0.2.1"
-
-[target.'cfg(target_os = "linux")'.dependencies]
-notify-rust = "4"
+notify-rust = "4.5.6"
 
 [profile.release]
 lto = true

--- a/src/main.rs
+++ b/src/main.rs
@@ -24,8 +24,6 @@ use simplelog::{ColorChoice, Config as LogConfig, LevelFilter, TermLogger, Termi
 use std::error::Error;
 use std::time::Duration;
 use tokio::task::JoinHandle;
-
-#[cfg(target_os = "linux")]
 use notify_rust::Notification;
 
 mod color;
@@ -215,7 +213,6 @@ async fn run() -> Result<Option<JoinHandle<()>>, Box<dyn Error>> {
                                 // Won't be executed when exec was successful
                                 error!("{}", err);
 
-                                #[cfg(target_os = "linux")]
                                 Notification::new()
                                     .summary("Kickoff")
                                     .body(&format!("{}", err))

